WFP-VAM/prism-app
# Fix layer group sub selection when toggling a layer back on

### Description

This fixes #1078 .



## How to test the feature:

- select a layer with group selection and change it to eg. 1 month
- switch between layers check that the loaded layer and the label are consistent 

## Checklist - did you ...



Test your changes with


## Screenshot/video of feature:
